src/GenAPI/Microsoft.DotNet.GenAPI/CSharpFileBuilder.cs

```diff
@@ -150,7 +142,7 @@ private static bool SignatureEquals(IMethodSymbol? method, IMethodSymbol? baseMe
                 return true;
             }
 
-            if (GetNonInterfaceName(method.Name) != GetNonInterfaceName(baseMethod.Name))
+            if (method.Name != baseMethod.Name)
```

Comment:
Did you want to include a test case for implicit vs explicit interface implementation to ensure `new` is not added?

Reply:
Yeah I believe the `IFun` + `Fun` method is the test case that checks for this.

Reply:
lemme add one for explicit implementation